kokoro/ubuntu/new_release.sh

```diff
@@ -29,10 +30,14 @@ cd git/google-cloud-eclipse
 #
 # https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2363#issuecomment-327844378
 # https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/2211
-readonly CT4E_DISPLAY_VERSION=$( \
-  xmlstarlet sel -t -v '/product/@version' gcp-repo/metadata.product )
+readonly CT4E_DISPLAY_VERSION="$( \
```

Comment:
Might be worth a comment on what this is doing: retrieve the product version, which we assume either has a `.qualifier` or no version qualifier, and append the provided build qualifier (if any).